test/unit/org/apache/cassandra/tools/nodetool/stats/TableStatsPrinterTest.java

```diff
@@ -507,6 +595,14 @@ public class TableStatsPrinterTest extends TableStatsTestBase
         "\tAverage tombstones per slice (last five minutes): 6.00\n" +
         "\tMaximum tombstones per slice (last five minutes): 6\n" +
         "\tDroppable tombstone ratio: 0.667\n" +
+        "\tSAI local query latency (mean): 0.000 ms\n" +
+        "\tSAI post-filtering latency (mean): 0.000 ms\n" +
+        "\tSAI space used (bytes): null\n" +
+        "\tSAI SSTable indexes hit per query (mean): 0.0\n" +
+        "\tSAI index segments hit per query (mean): 0.0\n" +
+        "\tSAI rows filtered per query (mean): 0.0\n" +
+        "\tSAI local total query timeouts: 0\n" +
+        "\tSAI queryable/total indexes: null\n" +
```

Comment:
We should have a few of these test tables present non-zero/null values. Here are some reasonable examples:

```
SAI local query latency (mean): 10.000 ms
SAI post-filtering latency (mean): 1.000 ms
SAI space used (bytes): 128000000
SAI SSTable indexes hit per query (mean): 3.5
SAI index segments hit per query (mean): 4.0
SAI rows filtered per query (mean): 55.0
SAI local total query timeouts: 4
SAI queryable/total indexes: 5/5
```

Reply:
I added test table5 with SAI metrics.